src/app/(unauth)/roadmap/components/RoadmapColumn.tsx

```diff
@@ -5,12 +5,13 @@ import { useCallback, useEffect, useState } from "react";
 import FeedbackCard from "./FeedbackCard";
 import { classNames } from "@/lib/utils";
 import { useRouter } from "next/navigation";
+import { Draggable } from "@hello-pangea/dnd";
 
 type RoadmapColumnPropsType = {
   status: string;
   projectsId: string;
   feedbackStatusMap: { [key: string]: FeedbackPostType[] };
-  setFeedbackStatusMap: (map: { [key: string]: FeedbackPostType[] }) => void;
+  setFeedbackStatusMap: any;
```

Comment:
try to use types of variable. ignore using the any

Reply:
ok